File: avro4k/src/main/java/Main.java
import data.Student;

import java.util.Arrays;

import static data.Serializer.serializeStudent;

public class Main {
    public static void main(String[] args) {
        Student student = new Student("Marius", 12, Arrays.asList(2, 6, 5), null);
        String text = serializeStudent(student);
        System.out.println(text);
    }
}

File: avro4k/src/main/kotlin/Main.kt
import com.github.avrokotlin.avro4k.Avro
import com.github.avrokotlin.avro4k.io.AvroEncodeFormat
import data.Student
import java.io.ByteArrayOutputStream

fun main() {
    val schema = Avro.default.schema(Student.serializer())
    println(schema.toString(true))
}